File: src/hashmap/lc_1347_minimum_steps_to_make_two_strings_anagram/minimum_steps_to_make_two_strings_anagram.py
from collections import defaultdict
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        counts = defaultdict(int)
        for ch in s:
            counts[ch] += 1
        steps = 0
        for ch in t:
            if ch in counts and counts[ch] > 0:
                counts[ch] -= 1
        
        for key in counts:
            steps += counts[key]
        return steps

File: src/hashmap/lc_1347_minimum_steps_to_make_two_strings_anagram/minimum_steps_to_make_two_strings_anagram.go
func minSteps(s string, t string) int {
    charMaps := make(map[rune]int)
    for _, char := range s {
        charMaps[char]++
    }
    for _, char := range t {
        if count, ok := charMaps[char]; ok && count > 0 {
            charMaps[char]--
        }
    }
    steps := 0
    for _, count := range charMaps {
        steps += count
    }
    return steps
}